inc/utils/content.php

```diff
@@ -564,17 +581,19 @@ function get_categories_array($post){
 
  function odm_echo_extras($postID = "") {
  	 $postID = $postID ? $postID : get_the_ID();
-	 if (function_exists('get_post_meta')) :
-		 $get_author = get_post_meta($postID, 'author', true);
-		 $get_localized_author = get_post_meta($postID, 'author_'.odm_language_manager()->get_current_language(), true);
-	   if ($get_author != '' || $get_localized_author != ''):
-	     $news_source_info = '<span class="lsf">&#xE041;</span> ';
-	     if ($get_localized_author != ''):
-	         $news_source_info .= $get_localized_author.'<br />';
-	     else:
-	         $news_source_info .= $get_author.'<br />';
-	     endif;
-	   endif;
+	 if (function_exists('get_post_meta')):
```

Comment:
> I would suggest that there be an agreed list of classifications/categories, if there isn't already.

@DBishton yes, the announcements content type on ODC has a custom taxonomy called **Public announcement Sources** which represents a finite list of options for editors to choose: https://pp.opendevelopmentcambodia.net/wp-admin/edit-tags.php?taxonomy=public_announcement_source&post_type=announcement

The two red areas you marked on the left half of your screenshot are the News Sources which are also a custom taxonomy for news articles content type: https://pp.opendevelopmentcambodia.net/wp-admin/edit-tags.php?taxonomy=news_source&post_type=news-article